library/ix-dev/community/netbootxyz/templates/_service.tpl

```diff
@@ -0,0 +1,35 @@
+{{- define "netboot.service" -}}
+service:
+  netboot:
+    enabled: true
+    primary: true
+    type: NodePort
+    targetSelector: netboot
+    ports:
+      tftp:
+        enabled: true
+        primary: true
+        port: {{ .Values.netbootNetwork.tftpPort }}
+        nodePort: {{ .Values.netbootNetwork.tftpPort }}
+        targetPort: 69
+        protocol: udp
+        targetSelector: netboot
+      assets:
+        enabled: true
+        port: {{ .Values.netbootNetwork.webAssetsPort }}
+        nodePort: {{ .Values.netbootNetwork.webAssetsPort }}
+        targetPort: {{ .Values.netbootConfig.NGINX_PORT }}
+        targetSelector: netboot
+  netboot-portal:
+    enabled: true
+    type: NodePort
+    targetSelector: netboot
+    ports:
+      portal-http:
+        enabled: true
+        primary: true
+        port: {{ .Values.netbootNetwork.webHttpPort }}
+        nodePort: {{ .Values.netbootNetwork.webHttpPort }}
+        targetPort: 3000
```

Comment:
It would be nice we could dynamically change this like nginx port, but thats not a blocker.
(Mostly for users that might have something else on port 3000 to avoid conflicts. Chances are low)